core/src/main/resources/reference.conf

```diff
@@ -97,7 +97,12 @@ workflow-options {
 
 // Optional call-caching configuration.
 call-caching {
+  # Allows re-use of existing results for jobs you've already run
```

Comment:
Document default values here and below.  The defaults might be a little surprising given that this config explicitly sets them to their default values.